src/application-status-overlay/ApplicationStatusOverlay.jsx

```diff
@@ -0,0 +1,69 @@
+import { useRef, useContext, useLayoutEffect } from 'react';
+import PropTypes from 'prop-types';
+import Button from 'terra-button';
+import uuidv4 from 'uuid/v4';
+import ApplicationStatusOverlayContext from './ApplicationStatusOverlayContext';
+
+const propTypes = {
+  /**
+   * An array of objects containing terra-button properties. A key attribute is required for each object.
+   * This array is used to render buttons in the bottom section.
+   * Example:`[{ text: 'Button 1', key: 1, size: 'medium', variant: 'action', onClick: onClickFunction}]`
+   */
+  // eslint-disable-next-line react/forbid-foreign-prop-types
+  buttonAttrs: PropTypes.arrayOf(PropTypes.shape(Button.propTypes)),
+
+  /**
+   * The descriptive text, displayed under the title.
+   */
+  message: PropTypes.string,
+
+  /**
+   * The title displayed under the glyph. Variants contain default titles that can be overridden by this prop.
+   */
+  title: PropTypes.string,
```

Comment:
Pretty sure we wanted to no longer expose the `title` in this new implementation: 

https://github.com/cerner/terra-application/issues/30#issuecomment-637568575

Reply:
Removed the `title` prop : https://github.com/cerner/terra-application/pull/47/commits/6fca8b18fef599c004d9efd588517068495dd45f